UDV-Core/webpack.config.js

```diff
@@ -62,4 +66,9 @@ module.exports = {
     devServer: {
         publicPath: '/dist/',
     },
+    resolve: {
```

Comment:
Could you add a comment saying that this is related to alpaca.

Reply:
done